### ISSUE TITLE
[ENHANCEMENT] component upgrade after 1.1.2-rc4

### ISSUE THREAD

**Issue by bk201** [harvester/harvester#3732]
- [x] OS (https://github.com/harvester/harvester-installer/pull/462)
- [x] KubeVirt (https://github.com/harvester/harvester/pull/3728)

**Comment by TachunLin** [comment]
* Check the base OS bump to `20230328`
```
harvester-node-0:~ # cat /etc/os-release
NAME="SLE Micro"
VERSION="5.3"
VERSION_ID="5.3"
PRETTY_NAME="Harvester v1.1.2-rc5"
ID="sle-micro-rancher"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sle-micro-rancher:5.3"
VARIANT="Harvester"
VARIANT_ID="Harvester-20230328"
GRUB_ENTRY_NAME="Harvester v1.1.2-rc5"
```

* Check the KubeVirt image bump to `0.54.0-150400.3.10.4`
```
harvester-node-0:~ # kubectl get deployments -A -o jsonpath='{range .items[*]}{@.metadata.name}{" "}{@.spec.template.spec.containers[*].image}{"\n"}{end}'|column -t
...                                                                            
virt-api                               registry.suse.com/suse/sles/15.4/virt-api:0.54.0-150400.3.10.4                                                  
virt-controller                        registry.suse.com/suse/sles/15.4/virt-controller:0.54.0-150400.3.10.4                                           
virt-operator                          registry.suse.com/suse/sles/15.4/virt-operator:0.54.0-150400.3.10.4                                             
...
```

- [ ] VM operation
   - [x] Create 
   - [x] Delete 
   - [ ] Restart VM - **Failed**
   - [x] Soft reboot
   - [ ] Stop -> Start - **Failed**
   - [x] Pause 
   - [x] Edit 
   - [x] Add volume
- [x] VM migrate
- [x] VM backup to S3
- [x] VM restore from S3
- [x] VM Clone

**Comment by TachunLin** [comment]
Verified **failed** on `v1.1.2-rc5`. Move the issue to implement state

### Result
When we try to restart a running VM, it will keep in `Starting` state, just like stated in issue #3727 
![image](https://user-images.githubusercontent.com/29251855/228810023-6e1e1c90-7d48-45cb-a582-87626186d5f5.png)

The VM event display error `FailedMapVolume` error
![image](https://user-images.githubusercontent.com/29251855/228810384-4a2391a1-d25f-4de1-9081-b8ae50c79d7e.png)

### Test Information
* Test Environment: 4 nodes Harvester on kvm machines
* Harvester version: `v1.1.2-rc5`

### Verify Steps
1. Create a VM with any type of network 
2. Start the VM in `Running` state
3. Restart the VM

**Comment by lanfon72** [comment]
@bk201 when bump kubevirt, I think we should also check `virtctl` is compatiable.

![image](https://user-images.githubusercontent.com/5169694/228820444-600f04ec-7555-42f1-9fd3-a9d69a663f67.png)

**Comment by bk201** [comment]
@lanfon72 The newer version was chosen from upstream because we want more CVE fixes. We can do this within this issue: https://github.com/harvester/harvester/issues/3587; to ship the version from IBS/OBS. cc @Vicente-Cheng

**Comment by bk201** [comment]
@TachunLin We can close this one once the version matches. The starting issue can be tracked in https://github.com/harvester/harvester/issues/3727

**Comment by TachunLin** [comment]
Verified **fixed** after upgrade to `v1.1.2-rc6`.  Close this issue.

* Check the base OS bump to `20230406`
```
node1:~ # cat /etc/os-release
NAME="SLE Micro"
VERSION="5.3"
VERSION_ID="5.3"
PRETTY_NAME="Harvester v1.1.2-rc6"
ID="sle-micro-rancher"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sle-micro-rancher:5.3"
VARIANT="Harvester"
VARIANT_ID="Harvester-20230406"
GRUB_ENTRY_NAME="Harvester v1.1.2-rc6"
```

* Check the KubeVirt image bump to `0.54.0-150400.3.10.4`
```
node1:~ # kubectl get deployments -A -o jsonpath='{range .items[*]}{@.metadata.name}{" "}{@.spec.template.spec.containers[*].image}{"\n"}{end}'|column -t
...                                                                            
virt-api                               registry.suse.com/suse/sles/15.4/virt-api:0.54.0-150400.3.10.4                                                  
virt-controller                        registry.suse.com/suse/sles/15.4/virt-controller:0.54.0-150400.3.10.4                                           
virt-operator                          registry.suse.com/suse/sles/15.4/virt-operator:0.54.0-150400.3.10.4                                             
...
```

- [x] VM operation
   - [x] Create 
   - [x] Delete 
   - [x] Restart VM
   - [x] Soft reboot
   - [x] Stop -> Start
   - [x] Pause 
   - [x] Edit 
   - [x] Add volume
